[2018-10-12 09:19:32,484] - [MainThread] - [log.py-line:22] [DEBUG] 认证成功！
[2018-10-12 09:19:39,025] - [MainThread] - [log.py-line:24] [WARNING] (2003, "Can't connect to MySQL server on '127.0.0.1' ([WinError 10061] 由于目标计算机积极拒绝，无法连接。)")
[2018-10-12 09:20:13,506] - [MainThread] - [log.py-line:22] [DEBUG] +----+----------+-----+-------+---------+----------------------------+----------------------------+
| id | username | age |  tel  | address |        create_time         |        update_time         |
+----+----------+-----+-------+---------+----------------------------+----------------------------+
| 1  |    y1    |  88 | 13333 |    fj   | 2018-10-10 15:34:49.909826 | 2018-10-11 14:28:41.537136 |
+----+----------+-----+-------+---------+----------------------------+----------------------------+
[2018-10-12 09:20:13,508] - [MainThread] - [log.py-line:22] [DEBUG] 
[2018-10-12 09:20:16,606] - [MainThread] - [log.py-line:22] [DEBUG] +----+----------+-----+-----------+---------+----------------------------+----------------------------+
| id | username | age |    tel    | address |        create_time         |        update_time         |
+----+----------+-----+-----------+---------+----------------------------+----------------------------+
| 1  |    y1    |  88 |   13333   |    fj   | 2018-10-10 15:34:49.909826 | 2018-10-11 14:28:41.537136 |
| 6  |   yyy    |  22 |   13333   |    fj   | 2018-10-11 13:57:59.723340 | 2018-10-11 13:57:59.723340 |
| 7  |   yyp    |  33 |   13332   |  fujian | 2018-10-11 16:37:56.322232 | 2018-10-11 16:37:56.322232 |
| 8  |    li    |  55 | 153333333 | beijing | 2018-10-11 16:41:02.430314 | 2018-10-11 16:41:02.430314 |
+----+----------+-----+-----------+---------+----------------------------+----------------------------+
[2018-10-12 09:20:16,608] - [MainThread] - [log.py-line:22] [DEBUG] 
[2018-10-12 09:20:41,338] - [MainThread] - [log.py-line:24] [WARNING] ("'builtin_function_or_method' object has no attribute 'isdigit'",)
[2018-10-12 09:20:55,676] - [MainThread] - [log.py-line:24] [WARNING] ("'builtin_function_or_method' object has no attribute 'isdigit'",)
[2018-10-12 09:21:20,263] - [MainThread] - [log.py-line:24] [WARNING] ("'builtin_function_or_method' object has no attribute 'isdigit'",)
[2018-10-12 09:21:26,538] - [MainThread] - [log.py-line:22] [DEBUG] +----+----------+-----+-----------+---------+----------------------------+----------------------------+
| id | username | age |    tel    | address |        create_time         |        update_time         |
+----+----------+-----+-----------+---------+----------------------------+----------------------------+
| 1  |    y1    |  88 |   13333   |    fj   | 2018-10-10 15:34:49.909826 | 2018-10-11 14:28:41.537136 |
| 6  |   yyy    |  22 |   13333   |    fj   | 2018-10-11 13:57:59.723340 | 2018-10-11 13:57:59.723340 |
| 7  |   yyp    |  33 |   13332   |  fujian | 2018-10-11 16:37:56.322232 | 2018-10-11 16:37:56.322232 |
| 8  |    li    |  55 | 153333333 | beijing | 2018-10-11 16:41:02.430314 | 2018-10-11 16:41:02.430314 |
+----+----------+-----+-----------+---------+----------------------------+----------------------------+
[2018-10-12 09:21:26,539] - [MainThread] - [log.py-line:22] [DEBUG] 
[2018-10-12 09:21:38,455] - [MainThread] - [log.py-line:24] [WARNING] ("'builtin_function_or_method' object has no attribute 'isdigit'",)
[2018-10-12 09:22:16,737] - [MainThread] - [log.py-line:22] [DEBUG] 认证成功！
[2018-10-12 09:22:18,911] - [MainThread] - [log.py-line:22] [DEBUG] +----+----------+-----+-----------+---------+----------------------------+----------------------------+
| id | username | age |    tel    | address |        create_time         |        update_time         |
+----+----------+-----+-----------+---------+----------------------------+----------------------------+
| 1  |    y1    |  88 |   13333   |    fj   | 2018-10-10 15:34:49.909826 | 2018-10-11 14:28:41.537136 |
| 6  |   yyy    |  22 |   13333   |    fj   | 2018-10-11 13:57:59.723340 | 2018-10-11 13:57:59.723340 |
| 7  |   yyp    |  33 |   13332   |  fujian | 2018-10-11 16:37:56.322232 | 2018-10-11 16:37:56.322232 |
| 8  |    li    |  55 | 153333333 | beijing | 2018-10-11 16:41:02.430314 | 2018-10-11 16:41:02.430314 |
+----+----------+-----+-----------+---------+----------------------------+----------------------------+
[2018-10-12 09:22:18,914] - [MainThread] - [log.py-line:22] [DEBUG] 
[2018-10-12 09:22:23,039] - [MainThread] - [log.py-line:22] [DEBUG] 数据删除成功！
[2018-10-12 09:22:36,965] - [MainThread] - [log.py-line:22] [DEBUG] +----+----------+-----+-------+---------+----------------------------+----------------------------+
| id | username | age |  tel  | address |        create_time         |        update_time         |
+----+----------+-----+-------+---------+----------------------------+----------------------------+
| 1  |    y1    |  88 | 13333 |    fj   | 2018-10-10 15:34:49.909826 | 2018-10-11 14:28:41.537136 |
| 6  |   yyy    |  22 | 13333 |    fj   | 2018-10-11 13:57:59.723340 | 2018-10-11 13:57:59.723340 |
| 7  |   yyp    |  33 | 13332 |  fujian | 2018-10-11 16:37:56.322232 | 2018-10-11 16:37:56.322232 |
+----+----------+-----+-------+---------+----------------------------+----------------------------+
[2018-10-12 09:22:36,966] - [MainThread] - [log.py-line:22] [DEBUG] 
[2018-10-12 09:22:41,461] - [MainThread] - [log.py-line:24] [WARNING] 不存在该用户！
[2018-10-12 09:22:45,241] - [MainThread] - [log.py-line:22] [DEBUG] +----+----------+-----+-------+---------+----------------------------+----------------------------+
| id | username | age |  tel  | address |        create_time         |        update_time         |
+----+----------+-----+-------+---------+----------------------------+----------------------------+
| 1  |    y1    |  88 | 13333 |    fj   | 2018-10-10 15:34:49.909826 | 2018-10-11 14:28:41.537136 |
| 6  |   yyy    |  22 | 13333 |    fj   | 2018-10-11 13:57:59.723340 | 2018-10-11 13:57:59.723340 |
| 7  |   yyp    |  33 | 13332 |  fujian | 2018-10-11 16:37:56.322232 | 2018-10-11 16:37:56.322232 |
+----+----------+-----+-------+---------+----------------------------+----------------------------+
[2018-10-12 09:22:45,243] - [MainThread] - [log.py-line:22] [DEBUG] 
[2018-10-12 09:22:52,929] - [MainThread] - [log.py-line:22] [DEBUG] +----+----------+-----+-------+---------+----------------------------+----------------------------+
| id | username | age |  tel  | address |        create_time         |        update_time         |
+----+----------+-----+-------+---------+----------------------------+----------------------------+
| 1  |    y1    |  88 | 13333 |    fj   | 2018-10-10 15:34:49.909826 | 2018-10-11 14:28:41.537136 |
+----+----------+-----+-------+---------+----------------------------+----------------------------+
[2018-10-12 09:23:10,504] - [MainThread] - [log.py-line:22] [DEBUG] 数据更新成功！
[2018-10-12 09:23:13,465] - [MainThread] - [log.py-line:22] [DEBUG] +----+----------+-----+-------+---------+----------------------------+----------------------------+
| id | username | age |  tel  | address |        create_time         |        update_time         |
+----+----------+-----+-------+---------+----------------------------+----------------------------+
| 1  |    li    |  33 | 13333 |    fj   | 2018-10-10 15:34:49.909826 | 2018-10-12 09:23:10.499508 |
| 6  |   yyy    |  22 | 13333 |    fj   | 2018-10-11 13:57:59.723340 | 2018-10-11 13:57:59.723340 |
| 7  |   yyp    |  33 | 13332 |  fujian | 2018-10-11 16:37:56.322232 | 2018-10-11 16:37:56.322232 |
+----+----------+-----+-------+---------+----------------------------+----------------------------+
[2018-10-12 09:23:13,467] - [MainThread] - [log.py-line:22] [DEBUG] 
[2018-10-12 09:23:26,403] - [MainThread] - [log.py-line:22] [DEBUG] 成功保存用户信息至xls文件！
[2018-10-12 09:23:35,193] - [MainThread] - [log.py-line:24] [WARNING] 不存在输入的操作，请重试！
[2018-10-12 09:23:39,054] - [MainThread] - [log.py-line:22] [DEBUG] 正在保存数据至excel文件！
[2018-10-12 09:23:39,064] - [MainThread] - [log.py-line:22] [DEBUG] 正在退出系统！
